src/Transform/Argify.hs

```diff
@@ -50,20 +48,9 @@ argify (TmElimProd am ptm ps tm tp) = TmElimProd am (argify ptm) ps (argify tm)
 argify (TmEqs tms) = TmEqs (argify <$> tms)
 
 argifyProg :: Prog -> Prog
-argifyProg (ProgFun x [] tm tp) =
-  let tm' = argify tm
-      (as, etp) = splitArrows tp
-      (ls, etm) = splitLams tm'
-      etas = [ (etaName x i, atp) | (i, atp) <- drop (length ls) (enumerate as) ]
-      etm_eta = joinApps etm (paramsToArgs etas)
-      ls_eta = ls ++ etas
-  in
-    ProgFun x ls_eta etm_eta etp
-argifyProg (ProgExtern x [] tp) =
-  let (tps, etp) = splitArrows tp in
-    ProgExtern x tps etp
+argifyProg (ProgFun x tp tm) = ProgFun x tp (argify tm)
```

Comment:
Are the comments at the top of the file still accurate?

Reply:
I don't think the first comment is accurate:

```
1. define foo = \ a. \ b. tm;   =>   define foo a b = tm;
```

because we don't even allow the second syntax and I am not sure when we ever did.

Reply:
Oh, I see, yes, that first comment used to be true in a sense, but no longer (AffLin just uses splitArrows/joinArrows now.) I will edit it.